utils.zsh/functions/check_cross

```diff
@@ -0,0 +1,23 @@
+autoload -Uz log_info log_status log_error log_debug log_warning
+
+log_debug 'Checking for apt-get...'
+if (( ! ${+commands[apt-get]} )) {
+  log_error 'No apt command found - please install apt'
```

Comment:
'No apt-get command found. Please install apt.'

While apt-get is the command, apt is the package.

Though, I have no idea how that would occur.

Reply:
It should not, but the check is more "complete" that way.

Reply:
Sure. I don't mind keeping the check. Resolve once the string change is addressed.